### PR TITLE
WV-2094: Tracking Data Collections 

### PR DESCRIPTION
**Pull request by ryanweiler92**
## Description

This adds a collection identification badge for layers that have been fully transitioned from GIBS to GITC. The badge will identify if the layer is currently utilizing NRT or Standard data and the version of the collection.

## How To Test
1. `git checkout tracking-data-collection-dates-update`
2. `npm ci`
3. `npm run watch`
4. [Use this link to load layers that have been transitioned to GITC.](http://localhost:3000/?v=-218.12488563727786,-136.40800990028663,148.59126442506462,149.963965466579&l=Reference_Labels_15m,Reference_Features_15m(hidden),Coastlines_15m,VIIRS_SNPP_Aerosol_Type_Deep_Blue_Best_Estimate,VIIRS_NOAA20_Cloud_Optical_Thickness,VIIRS_NOAA20_Clear_Sky_Confidence_Day,VIIRS_CrIS_SNPP_BT_Band33_Fusion_Day,BlueMarble_NextGeneration(hidden),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=false&t=2022-12-12-T17%3A52%3A53Z)
5. Verify that the following 4 layers have the collection identifier badge 1. Deep Blue Aerosol Type 2. Cloud Optical Thickness 3. Clear Sky Confidence (Day) 4. Band 33 Brightness Temperature. 
6. Increment the date day by day until you get to the current date. The layers should eventually change to NRT or display no badge at all.

### CODE REVIEW

**Review by minniewong — APPROVED** [review]
Looks really good!

**Comment by Tomcariello** [comment]
I am seeing this warning in the console. Can we address it?
`Warning: Failed prop type: Invalid prop `delay` supplied to `Tooltip`, expected one of type [number, number].`

File: web/js/containers/sidebar/layer-row.js [hunk 1]
```diff
@@ -351,13 +357,27 @@ function LayerRow (props) {
 
       <Zot zot={zot} layer={layer.id} isMobile={isMobile} />
 
-      <div className={isVectorLayer ? 'layer-main wv-vector-layer' : 'layer-main'}>
+      <div id="test" className={isVectorLayer ? 'layer-main wv-vector-layer' : 'layer-main'}>
```
Comment (Tomcariello) [inline web/js/containers/sidebar/layer-row.js:360]: Can we remove this "test" id?

File: web/js/modules/layers/actions.js [hunk 2]
```diff
@@ -391,3 +393,24 @@ export function changeGranuleSatelliteInstrumentGroup(id, granulePlatform) {
     });
   };
 }
+
+export function updateLayerCollection(value) {
```
Comment (Tomcariello) [inline web/js/modules/layers/actions.js:397]: It is unclear to me what data "value" represents here. Can we provide a function definition? Or perhaps we can just pass the id into this function instead?

File: web/js/modules/layers/actions.js [hunk 3]
```diff
@@ -391,3 +393,24 @@ export function changeGranuleSatelliteInstrumentGroup(id, granulePlatform) {
...
+    if (!collections) {
+      dispatch({
+        type: UPDATE_LAYER_COLLECTION,
+        id,
+      });
+    }
+  };
+}
+
+export function updateLayerDateCollection(value) {
```
Comment (Tomcariello) [inline web/js/modules/layers/actions.js:411]: Same as above. Can we provide a function definition?

File: web/js/containers/sidebar/layer-row.js (hunk 1 above)
Comment (ryanweiler92) [reply web/js/containers/sidebar/layer-row.js]: Removed this id.

File: web/js/modules/layers/actions.js (hunk 2 above)
Comment (ryanweiler92) [reply web/js/modules/layers/actions.js]: I've changed this to `id` and extracted the id from the object when we dispatch the action in `layerbuilder.js`.

File: web/js/modules/layers/actions.js (hunk 3 above)
Comment (ryanweiler92) [reply web/js/modules/layers/actions.js]: Renamed this parameter to `layerInfo`

**Review by ZachTRice — APPROVED** [review]

**Review by Tomcariello — APPROVED** [review]
Looks good!

**Pull request merged**